student/views.py

```diff
@@ -1,21 +1,22 @@
 from django.shortcuts import render, redirect, get_object_or_404
+
+import jobboard.views
 from .models import Student
 from django.contrib import messages
 from .forms import UpdateStudentAccountSettingsForm, StudentRegistrationForm
 
 
 def account_update_success(request):
     context = {}
-    add_is_student_to_context(request, context)
+    jobboard.views.add_navbar_links_to_context(request, context)
     return render(request, 'account_settings_update_success.html', context)
 
 
 def update_student_account_settings_view(request):
     context = {}
     student_object = get_object_or_404(Student, user_id=request.user.id)
-    update_student_form = UpdateStudentAccountSettingsForm(
-        request.POST or None, instance=student_object)
-    add_is_student_to_context(request, context)
+    update_student_form = UpdateStudentAccountSettingsForm(request.POST or None, instance=student_object)
```

Comment:
Can you please explain this line?

Reply:
@guy9050 
It makes the form handle both a POST request and a GET request, I can change it if you want but this code is already merged and working and not really related to this PR. I just deleted some spaces around it this is why it looks like it's new.